CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Mattermost Desktop Application Changelog
 
+## Release v3.7.1
+
+Release date: August 30, 2017
+
+This release contains a security update and it is highly recommended that users upgrade to this version.
```

Comment:
Do we need to explicitly mention what the update was? As it stands, I see no explanation of why this is an important update.

Reply:
The security update is not only for Windows, but our changelog contains sentences only for Windows. Should we tell about this point?